docs/examples.rst

```diff
@@ -20,4 +20,5 @@ Examples
   notebooks/negative_binomial
   notebooks/beta_regression
   notebooks/categorical_regression
-      notebooks/shooter_crossed_random_ANOVA
+  notebooks/shooter_crossed_random_ANOVA
```

Comment:
I think this example does not work, that's why it's not included.

It's going to work again once we use formulae 0.3.0 (it needs to be adapted a little though)